dpm/tests/bdd/steps/properties_secrets_step.py

```diff
@@ -32,9 +37,6 @@ def __init__(self):
 
 
 class SecretManagerServiceClientMock(Mock):
-    def list_secret_versions(self, x):
```

Comment:
Duplicate method found by MyPy pre-commit hook.

Reply:
Hmm, I'm wondering how this was detected? It seems that the return here is a List of a List of MockObj's. 

(Could be from another location I'm not seeing in this review page)

Reply:
The other method of the same name is a few methods below this and returns a mock itself which can mimic a list to the caller already.  MyPy detected this as a duplicate method as neither had typed signatures so it then falls back to name checking.